src/packages/sdk/src/getGenerators.ts

```diff
@@ -488,7 +500,7 @@ async function validateGenerators(
   const platform = await getPlatform()
 
   for (const generator of generators) {
-    if (generator.provider === 'photonjs') {
+    if (parseEnvValue(generator.provider) === 'photonjs') {
```

Comment:
Why is it needed here?

Reply:
https://prisma-company.slack.com/archives/C4GCG53BP/p1617008193084400